api/src/task/notifications/search_notification.py

```diff
@@ -101,6 +166,76 @@ def collect_email_notifications(self) -> list[UserEmailNotification]:
 
         return users_email_notifications
 
+    def _build_notification_message(self, opportunities: Sequence[Opportunity]) -> str:
+
+        # Build message intro based on number of opportunities
+        if len(opportunities) == 1:
+            message = (
+                "A funding opportunity matching your saved search query was recently published.\n\n"
+            )
+        else:
+            message = "The following funding opportunities matching your saved search queries were recently published.\n\n"
+
+        # Add details for each opportunity
+        for opportunity in opportunities:
+            summary = (
+                opportunity.current_opportunity_summary.opportunity_summary
+                if opportunity.current_opportunity_summary
+                else None
+            )
+
+            if not summary:
+                continue
+
+            # Add opportunity title (empty line before title)
+            message += f"{opportunity.opportunity_title}\n\n"
+
+            # Add status
+            status = (
+                str(opportunity.opportunity_status).capitalize()
+                if opportunity.opportunity_status
+                else "Unknown"
+            )
```

Comment:
FYI - if it has a summary it has a status since the field isn't nullable on the `current_opportunity_summary` object, but MyPy might not be able to recognize that so it's fine to leave this.